build_test.go

```diff
@@ -8,8 +8,13 @@ import (
 	"path/filepath"
 	"strings"
 	"testing"
+	"time"
 )
 
+// testBuildTimeout is timeout value constant used throughout tests
+// USED IN: build_test, execute_test, github_archive_test

```

Comment:
<personalOpinion> this comment can only ever cause problems.  I can use tooling to find where this constant is used.  If the real usage doesnt match the comment, what does that mean?  Should I update the comment or update the tests that use it?

I wouldnt have comments at all on this constant.
</personalOpinion>

Regardless you should move it below the copyright.